AcadeMeData/tests/test_msgboard_feature.py

```diff
@@ -0,0 +1,35 @@
+import pytest
+from AcadeMeData.forms import MessageForm
+from django.contrib.messages import get_messages
+
+
+@pytest.mark.django_db
+def test_msgboard_login(client, user_example):
+    client.force_login(user_example.user)
+    response = client.get("/msgboard/")
+    assert response.status_code == 200
+
+
+@pytest.mark.django_db
+def test_msgboard_form_instance(client, user_example):
+    client.force_login(user_example.user)
+    response = client.get("/msgboard/")
+    form = response.context["form"]
+    assert isinstance(form, MessageForm)
+
+
+@pytest.mark.django_db
+def test_msgboard_empty_message_when_page_loads(client, user_example):
+    client.force_login(user_example.user)
+    response = client.get("/msgboard/")
+    messages = list(get_messages(response.wsgi_request))
+    assert len(messages) == 0
+
+
+@pytest.mark.django_db
+def test_msgboard_feature(client, user_example, generate_msgboard):
```

Comment:
I would add here validation of the following
      assert len(response.context['messages']) > 0
      and also search for the content

Reply:
sorry you are avoiding the reviews, again, check that you have returned messages, you should not check the form
Not instance, content

Reply:
nvm, at least test were added